tests/ert/unit_tests/config/test_ert_config.py

```diff
@@ -2025,7 +2025,7 @@ def test_design2params_also_validates_design_matrix(tmp_path, caplog, monkeypatc
     assert "DESIGN_MATRIX validation of DESIGN2PARAMS" in caplog.text
 
 
-def test_multiple_design2params_also_validates_design_matrix_merging(
+def test_two_design2params_also_validates_design_matrix_merging(
```

Comment:
I guess it is enough to have just `test_three_design2params_also_validates_design_matrix_merging` without the also in the name.

Reply:
fixed